packages/ra-core/src/dataProvider/useGetManyAggregate.spec.tsx

```diff
@@ -85,6 +85,23 @@ describe('useGetManyAggregate', () => {
         expect(dataProvider.getMany).toHaveBeenCalledTimes(1);
     });
 
+    it('should not call dataProvider.getMany() if ids is empty', async () => {
+        const { rerender } = render(
+            <CoreAdminContext dataProvider={dataProvider}>
+                <UseGetManyAggregate resource="posts" ids={[]} />
+            </CoreAdminContext>
+        );
+        await new Promise(resolve => setTimeout(resolve));
+        expect(dataProvider.getMany).toHaveBeenCalledTimes(0);
```

Comment:
I'm curious about the value of data and isLoading in that case. Does that mean that the result is always loading that would be false, as the dataProvider is not even called.

Reply:
Also, if there are more than one request to useGetManyAggregate and one has an empty array, does it block the aggregation?